geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -55,12 +59,34 @@ public void sdiffErrors_givenTooFewArguments() {
   }
 
```

Comment:
It would be good to add an SDIFF version of the test `testSDiffStore_withNonSetKey()` in this class.

Reply:
Like a key that is used for another data structure type. IE. a key that is in sorted sets?

Reply:
Using a sorted set is a bit iffy, because native Redis does support doing diff/union/inter of a mixture of sorted sets and sets, but we don't currently support that in geode-for-redis. It'd be best to use a string, something like:
```
  @Test
  public void sdiff_withNonSetKey_throwsWrongtypeError() {
    jedis.sadd("{user1}set1", "member");
    jedis.set("{user1}string1", "value");

    assertThatThrownBy(() -> jedis.sdiff("{user1}set1", "{user1}string1"))
        .hasMessage("WRONGTYPE Operation against a key holding the wrong kind of value");
  }
```